requirements-dev.txt

```diff
@@ -6,4 +6,5 @@ pytest-xdist>=2.5.0
 flaky>=3.7.0
 pytest-forked>=1.4.0
 black>=21
-requests~=2.28.1
\ No newline at end of file
+requests~=2.28.1
+tomli~=2.0.0 # Drop once minimum Python version is 3.11
```

Comment:
In general, deps in this file are not already in other requirement files (`requests` shouldn't be here, oversight by me in a previous review), but then again this is a very dev-facing requirement so I'm fine with this duplication to make a statement. ok I guess ignore this message but here's some context for readers

Reply:
This was added from another PR during a merge. Fine to remove it from me as I don't need it here.